crates/astria-sequencer/src/service/mempool.rs

```diff
@@ -82,6 +84,14 @@ async fn handle_check_tx<S: StateReadExt + 'static>(
     let request::CheckTx {
         tx, ..
     } = req;
+    if tx.len() > MAX_TX_SIZE {
+        return response::CheckTx {
```

Comment:
This should contain the `info` line, which would be the `Display` impl of `AbciCode::TX_TOO_LARGE` (see comment below adding more abci codes). I think the Display line should be more akin to what's currently in log (see the comment in `log` for more)